service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/AbstractEmailContentProvider.java

```diff
@@ -73,12 +75,9 @@ protected AbstractEmailContentProvider(String uiBaseUrl,
     }
 
     private String getHearingBooking(CaseData data) {
-        if (!isNull(data.getHearingDetails())) {
-            return dateFormatterService.formatLocalDateToString(
-                hearingBookingService.getMostUrgentHearingBooking(
-                    data.getHearingDetails()).getStartDate().toLocalDate(), FormatStyle.LONG);
-        }
-        return "";
+        return hearingBookingService.getFirstHearing(data.getHearingDetails())
+            .map(hearing -> formatLocalDateToString(hearing.getStartDate().toLocalDate(), FormatStyle.LONG))
```

Comment:
Maybe not for this PR, but thoughts on moving away from the FormatStyle constants, it's not the most clear what format they are specifying

Reply:
I am not fussed either way, but we can discuss

Reply:
I just find the docs for those styles very wishy washy as it all comes down to the locale that is being used as well (I think). The docs basically say that it might be "some example".